timetracker/entries/templates/projects.html

```diff
@@ -12,8 +12,17 @@
   <ul class="list-group col-md-3">
     {% for project in project_list %}
       <li class="list-group-item">
-        <a href="{% url 'project-detail' pk=project.pk %}">{{ project.name }}</a> (<a href="{% url 'client-detail' pk=project.client.pk %}">{{ project.client.name }}</a>)
-        <a class="btn btn-warning btn-xs pull-right" href="{% url 'project-detail' pk=project.pk %}">Edit</a>
+        {% if project.author == request.user %}<a href="{% url 'project-detail' pk=project.pk %}">{% endif %}
```

Comment:
Is there a way to omit the <a> and </a> tags conditionally that I just don't know about? That is, without writing a custom template tag / filter?